pos_operating_unit_sequence/models/pos_order.py

```diff
@@ -0,0 +1,13 @@
+from odoo import models
+
+
+class PosOrder(models.Model):
+    _inherit = "pos.order"
+
+    def write(self, vals):
```

Comment:
In the original function we have the following:

`@api.model
  def create(self, values):
  session = self.env['pos.session'].browse(values['session_id'])
  values = self._complete_values_from_session(session, values)
  return super(PosOrder, self).create(values)`

`  @api.model
    def _complete_values_from_session(self, session, values):
    if values.get('state') and values['state'] == 'paid':
        values['name'] = session.config_id.sequence_id._next()
    values.setdefault('pricelist_id', session.config_id.pricelist_id.id)
    values.setdefault('fiscal_position_id', session.config_id.default_fiscal_position_id.id)
    values.setdefault('company_id', session.config_id.company_id.id)
    return values
`
  
Can we not simply inherit _complete_values_from_session to manipulate the name? 

In general i am wondering, if we need this module at all. 
Each pos-session clearly has a pos config in which we can define a sequence.